.travis.yml

```diff
@@ -1,17 +1,12 @@
 language: csharp
 sudo: false
-mono:
-  - 4.6.2 # in most common distros
-  - latest
-os:
-  - linux
-  - osx
 matrix:
-  exclude:
-    - os: osx
+  include:
+    - os: linux
+      mono: 4.6.2
+    - os: linux
       mono: latest
-  allow_failures:
-    - mono: latest
-  fast_finish: true
+    - os: osx
+      mono: 4.6.2
```

Comment:
I know this isn't a change in this PR - but do you know why we test osx on an old version? I thought OSX mono isn't bundled, so it makes more sense to test against latest stable?